ddtrace/tracer/writer.go

```diff
@@ -104,6 +104,16 @@ func (h *agentTraceWriter) flush() {
 	h.payload = newPayload()
 }
 
+var logWriter io.Writer = os.Stdout
+
+// setLogWriter sets the io.Writer that any new logTraceWriter will write to and returns a function
+// which will return the io.Writer to its original value.
+func setLogWriter(w io.Writer) func() {
```

Comment:
I think this func is only needed in tests right?

Reply:
I considered it, but since `var logWriter` is declared and used here, I thought the function for manipulating it should be co-located so the intention/usage is apparent for whoever next looks at `var logWriter`.

Let me know if you feel strongly that it should be moved.

Reply:
I think it's test code (unused elsewhere), so it belongs in a test. But likewise, if you feel strongly, leave it :)